io.github.jonmagon.kdiskmark.yaml

```diff
@@ -0,0 +1,43 @@
+app-id: io.github.jonmagon.kdiskmark
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+rename-desktop-file: kdiskmark.desktop
+rename-icon: kdiskmark
+copy-icon: true
+command: kdiskmark
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=host
+  - --device=dri
```

Comment:
you probably need to ensure you allow the dbus name, unless it got changed too to match the app-id.